### DISCUSSION TITLE
Koriki 1.5 won't boot on Miyoo Mini Plus

### DISCUSSION THREAD

**Discussion by joshkim-wustl** [Rparadise-Team/Koriki#44] [General]
I've formatted a new SD card to FAT32, then copied all the contents of Koriki 1.5 Full into the root of the SD card. When I insert the card into the MM+, I get the install message and the device shuts off as expected. When I try to restart it, it gives me the Koriki 1.5 screen, then shows me the black and white Koriki logo. The screen goes black, then goes back to the black and white logo. It keeps repeating this and I can't start up the OS at all. I saw someone else had the same problem on Reddit, but it looks like it was never resolved. Anyone know how to fix this? Thank you!

EDIT: I am on firmware version 202305052130

**Comment by ikentrock** [comment]
I'm having the same issue :(

https://github.com/user-attachments/assets/11433f71-60ef-43e5-90db-cb127f2a3be9

**Reply by Ninoh-FOX** [reply]
Can you press B button in the boot process?

**Reply by ikentrock** [reply]
yeap, I left it pressed in several stages of the startup: Miyoo screen, BOM gameboy and BOM koriki screens, same result :(

**Reply by Ninoh-FOX** [reply]
Erase last_state.sav for ./simplemenu folder and try again

**Reply by ikentrock** [reply]
I don't have that file in that route:

![image](https://github.com/user-attachments/assets/09089ba1-29dc-46d7-97a7-01a3252e8a5d)

But I have it in here: 
![image](https://github.com/user-attachments/assets/b7b25c2f-6607-48d0-932d-a3b34dabf626)

I removed it, and same situation :(

**Comment by Ninoh-FOX** [comment]
Try the img.gz file

**Comment by CarpeOmnia83** [comment]
Has anyone solved this? I copied Koriki to a new SD card formatted to FAT32. I did the set up with out incident then after reloading has been unable to get past the load screen. I have a Miyoo Mini Flip . Is this not supported? if not you should let people know because the rumor is that the flip is supported.

**Reply by Ninoh-FOX** [reply]
For Flip it's version 1.6.*, not 1.5, I've already seen you on Reddit, there's no need to write 30 messages saying it doesn't work if you don't know how to install it.
